### ISSUE TITLE
Distribution and installation

### ISSUE THREAD

**Issue by apjanke** [create-dmg/create-dmg#96]
Even though create-dmg is a near-trivial program, we should probably have a standard installation step (a `make install`, that is), which would make it play nicer with package managers, and make it a little easier for sysadmins to deal with installing. (Right now, they have to pick the location where it goes and manually copy selected files over.

And we should maybe have a canonical distribution tarball file, instead of just the archived source that git creates, to keep things clean.

If we do this, maybe we could remove some of the special Homebrew support that's in the codebase, because `brew` could treat this more like a normal application.

However this is done, it should not break the ability of `create-dmg` to be run directly from the source tree.

**Comment by apjanke** [comment]
Big question to figure out is what the directory structure of the installed application looks like, and how that will interact with the structure that's in the repo.

I'm thinking that the `support/` and `examples/` (and `doc/`, if we add that) folders should go under `$PREFIX/share/create-dmg`, and `create-dmg` itself should go under `$PREFIX/bin`. And then add some detection logic to the `AUX_PATH=...` code to see whether it's running from the repo or in the installed form.

Maybe we could add a man page, too.

**Comment by apjanke** [comment]
I've created a custom Homebrew tap at https://github.com/create-dmg/homebrew-create-dmg to work out the Homebrew installation issues on our side before we submit to Homebrew core.

**Comment by apjanke** [comment]
Okay, in https://github.com/create-dmg/create-dmg/commit/02c0b6bc0290d974cc934db11c8a123e1ea1cfaf I created a generic `make install` target and got rid of all the Homebrew-specific stuff. This is now working more like a "normal" Unix application. The main `create-dmg` script now has logic to detect the location of its `support/` folder, depending on whether it's running from within the Git repo or an installed location on the filesystem.

No need for additional distro tarballs; the source tree "archives" from GitHub are just fine because the `make install` target works correctly with them. `examples/`, `tests/`, and doco are installed under `$prefix/share`, and I've updated the example scripts to work correctly from both locations.

This is done.
